src/components/ui/Button/Button_sb.stories.js

```diff
@@ -18,16 +18,16 @@ export default {
             </div>
 
             <div className='flex space-x-2'>
-                <Button className='space-x-1'>
+                <Button className='space-x-1' data-size='small'>
                     <div>{BUTTON_TEXT} </div> <ArrowIcon className="text-white" />
                 </Button>
-                <Button className='space-x-1' variant="outline">
+                <Button className='space-x-1' variant="outline" data-size='medium'>
                     <div>{BUTTON_TEXT} </div> <ArrowIcon className="text-accent-900" />
                 </Button>
-                <Button className='space-x-1' variant="soft">
+                <Button className='space-x-1' variant="soft" data-size='large'>
```

Comment:
data-size attribute should be part of the button component

we only pass size prop when we use this component

Reply:
I didn’t use typescript before so I don’t know how to pass size prop in button component

Reply:
no issues, typescript is relatively easy to pick up, the best way to do is write a lot of typescript code

basically I'm saying, we just pass `size="small"` here, instead of data-size